tests/create_LWE.py

```diff
@@ -0,0 +1,242 @@
+from addict import Dict
+
+from CADETProcess.simulator import Cadet
+
+from CADETProcess.processModel import (
+
+    ComponentSystem,
+    FlowSheet,
+
+    Inlet,
+    Outlet,
+    Cstr,
+    GeneralRateModel,
+    TubularReactor,
+    LumpedRateModelWithoutPores,
+    LumpedRateModelWithPores,
+    MCT,
+
+    Linear,
+    Langmuir,
+    LangmuirLDF,
+    BiLangmuir,
+    BiLangmuirLDF,
+    FreundlichLDF,
+    StericMassAction,
+    AntiLangmuir,
+    Spreading,
+    SelfAssociation,
+    BiStericMassAction,
+    MultistateStericMassAction,
+    SimplifiedMultistateStericMassAction,
+    Saska,
+    GeneralizedIonExchange
+)
+
+
+def create_lwe(n_comp=4, unit_type=None, **kwargs) -> None:
+
+    def configure_all():
```

Comment:
[question] What's the reasoning behind making these functions local?

Reply:
These are just helper functions to either run a single LWE or LWEs for all unit types.